app.go

```diff
@@ -37,13 +51,13 @@ func (a *AppSettings) SetAPNConfig(c APNConfig) *AppSettings {
 	return a
 }
 
-func (a *AppSettings) SetFirebaseConfig(c FirebaseConfig) *AppSettings {
+func (a *AppSettings) SetFirebaseConfig(c FirebaseConfigRequest) *AppSettings {
```

Comment:
this separation is a good direction but beware that this is a breaking change

Reply:
Should we raise major? I don’t know how many people uses this

Reply:
I am fine with big major number as much as we're semantic